doc/user_guide/dynamo0p3.rst

```diff
@@ -156,7 +163,7 @@ Depending on the function space a field lives on, the field data at a
 point can be a scalar or a vector (see :ref:`dynamo0.3-function-space`
 for the list of scalar and vector function spaces). There is an
 additional option which specifies whether the data itself is vector
-valued and it usually refers to bundles of scalar valued fields.
+valued and it usually refers to bundles of scalar-valued fields.
```

Comment:
It means that field vectors can represent a bundle of either scalar- or vector-valued fields and in LFRic they are commonly used to represent  scalar-valued fields, so I wrote that explicitly.

Reply:
I've just commented in-line but I'll repeat myself here for continuity: thanks for explaining. Now that I know what it means I don't think we need it! Since this is documentation for PSyclone functionality, how frequently a given feature is used isn't important.

Reply:
As said elsewhere, I removed the bit about what is the common use in LFRic.